<launch>

    <group>
        <param name="gazebo_prefix" type="string" value="/gazebo" /> 


        <node pkg="icr" name="model_server" type="model_server" output="screen" ns="model_server">
            <param name="model_directory" type="string" value="$(find icr)/models/urdf/"/>
            <param name="reference_frame_id" type="string" value="/fixed"/>
            <remap from="set_object" to="/grasp_server/set_object"/> 
       </node>


    <node pkg="icr" name="Icr_Server" type="icr_server" output="screen" >
        <param name="icr_prefix" type="string" value="" />  
        <param name="alpha_shift" type="double" value="0.5" />
        <rosparam file="$(find icr)/input_grasps/test.yaml" command="load" ns="icr_fingers_param"/>		
    </node>

       <node pkg="icr" name="grasp_server" type="grasp_server" output="screen" ns="grasp_server">
           <param name="tf_prefix" type="string" value="/srh/position" />  
           <param name="spin_frequency" type="double" value="100" /> 
           <remap from="contact_points" to="/finger_tips"/> 
           <rosparam file="$(find icr)/config/grasp_server_config.yaml" command="load"/>
      </node>
  </group>

</launch>





